packages/ui/src/components/AddChats/AddDmSheet.tsx

```diff
@@ -64,33 +62,31 @@ export function StartDmSheet({
       <Sheet.Overlay />
       <Sheet.LazyFrame paddingTop="$s" paddingHorizontal="$2xl">
         <QueryClientProvider client={queryClient}>
-          <ContactsProvider contacts={contacts ?? null}>
```

Comment:
Confirmed removing the contact provider here breaks it on Android. Can't remember the exact specifics, but there's some bug with modal'd components not correctly getting context from further up in the tree

Reply:
Oof, ok. Was wondering why it was there. Will add it back w/a comment.